src/supermarket/Gemfile

```diff
@@ -69,7 +70,7 @@ group :development do
   gem "spring-commands-rspec"
   gem "brakeman"
 end
-
+gem "knife"
```

Comment:
What is knife being used for?

Reply:
This is removed .. since the PR was WIP it was pushed by mistake